orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -9200,6 +9216,40 @@ definitions:
     items:
       $ref: '#/definitions/metric_datapoint'
     type: array
+  mode_map_item:
+    description: Item containing {mode, [plmnA, plmB], [imsi1, imsi2], [apnY, apnZ]}
+    properties:
+      apns:
+        items:
+          example: internet.com
+          type: string
+          x-nullable: false
+        type: array
+      imsi_ranges:
+        items:
+          example: 000000000000000:000019999999999
+          type: string
+          x-nullable: false
+        type: array
+      mode:
+        default: federated_local
+        enum:
+        - federated_off
+        - federated_local
+        - federated_remote
+        example: federated_local
+        type: string
+        x-nullable: false
+      plmn_ids:
+        items:
+          example: "123456"
+          maxLength: 6
+          minLength: 5
+          pattern: ^(\d{5,6})$
+          type: string
+          x-nullable: false
+        type: array
+    type: object
```

Comment:
Maybe makes sense to simplify it a little:
1) avoid arrays of keys (imsi_range, apn, plmn_id)
2) rename federated_off & federated_remote
3) get rid of federated_local, it's a default/legacy case anyway